Lib/test/test_pickle.py

```diff
@@ -368,17 +392,20 @@ class SizeofTests(unittest.TestCase):
 
         def test_pickler(self):
             basesize = support.calcobjsize('6P2n3i2n3i2P')
+            P = struct.calcsize('P')
             p = _pickle.Pickler(io.BytesIO())
             self.assertEqual(object.__sizeof__(p), basesize)
             MT_size = struct.calcsize('3nP0n')
             ME_size = struct.calcsize('Pn0P')
             check = self.check_sizeof
             check(p, basesize +
+                2 * P +  # Managed dict
```

Comment:
I prefer it to be more aligned. like this
```
check(p, basesize + 
      2 * P + 
      ....
```
This suggestion applies to all changes.